src/.eslintrc

```diff
@@ -1,6 +0,0 @@
-{
-  "extends": [
-    "../.eslintrc",
-    "eslint-config-atomic/strict",
-  ]
-}
```

Comment:
Eslint-config-atomic already supports TypeScript and more underneath. I don't know why this is removed. 
https://github.com/atom-community/eslint-config-atomic/blob/master/src/typescript.cts

Reply:
1. It also pulls in a ton of other stuff (including conflicting peer dependencies)
2. `eslint-config-atomic` is failing CI and I suspect it will have trouble with eslint 9.
3. It's not clear *which* rules are in use and *why*. ZeroMQ is not part of the Atom project.

What particular rules from `eslint-config-atomic` am I missing that add value?